build.cmd

```diff
@@ -18,6 +18,12 @@ if %errorlevel% equ 0 (
   set outloop=true
 )
 
+echo %* | findstr /i /C:"FloatingTestRuntimeDependencies=true"  1>nul
+if %errorlevel% equ 0 (
+  set TestFixedRuntimeProjectJson=%~dp0src\Common\test-runtime\LatestDependencies\project.json
+  set defaultBuildConfig=/p:Configuration=Windows_NT_Debug
```

Comment:
@zhenlan I can understand it not being a goal of this PR, but I think being able to build WCF packages with latest (or just newer) package dependencies is a critical test scenario for us. How else can we know the packages we deliver will work for customers (prior to being deep in an RC release cycle)?  We can take this conversation off this PR and open a separate issue to discuss it if you think it is appropriate.  @hongdai -- don't block this PR on this discussion.